<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="test" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>741</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Config">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit_manualEnter">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>621</width>
        <height>71</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_English">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>390</y>
        <width>621</width>
        <height>61</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_source">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>621</width>
        <height>281</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="locale">
       <locale language="Chinese" country="China"/>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_submit">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>300</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Next">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>350</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Exit">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>430</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Content">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label_Chinese_File">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Chinese Text File</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Chinese_File">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_openChineseText">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_English_File">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_English_File">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>English Text File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_openEnglishText">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>853</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>test</class>
   <extends>QTabWidget</extends>
   <header>test.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>plainTextEdit_manualEnter</tabstop>
  <tabstop>pushButton_submit</tabstop>
  <tabstop>pushButton_Next</tabstop>
  <tabstop>pushButton_Exit</tabstop>
  <tabstop>plainTextEdit_English</tabstop>
  <tabstop>plainTextEdit_source</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_Chinese_File</tabstop>
  <tabstop>pushButton_openChineseText</tabstop>
  <tabstop>lineEdit_English_File</tabstop>
  <tabstop>pushButton_openEnglishText</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>719</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Next</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_English</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>651</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_submit</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_English</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Next</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_English</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>721</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
